### PR TITLE
Changes to saga text styles and null conversion

### LINKED ISSUE #642: Null values are not represented in SagaView
In the SagaView panels the null values are left empty on right hand side of the equal signs ("="). The lack of value should be clearly communicated. 

Options are using `<Null>`, `<Nil>` or `N/A`.

This is a split of #624.

Depends on Particular/ServiceControl#889

### PR DESCRIPTION
**Pull request by HEskandari**
Fixes #642

### CODE REVIEW

**Comment by HEskandari** [comment]
@Particular/serviceinsight-maintainers Please review.

File: src/ServiceInsight/Saga/SagaResources.xaml [hunk 1]
```diff
@@ -25,4 +25,13 @@
     
     <SolidColorBrush x:Key="FailedMessageBackground" Color="#F9916F" />
     <SolidColorBrush x:Key="RetriedMessageBackground" Color="#F2F2F2" />
+
+    <Style x:Key="GrayNullTextStyle" TargetType="{x:Type TextBlock}">
```
Comment (distantcam) [inline src/ServiceInsight/Saga/SagaResources.xaml]: This style doesn't make the text gray when it's null. Should the name be changed? Something more generic like `ValueTextStyle`

Comment (HEskandari) [inline src/ServiceInsight/Saga/SagaResources.xaml]: Yes, good point. It is more like 'ItalicWhenNull' style due to the Trigger. I'll rename it.

Comment (HEskandari) [inline src/ServiceInsight/Saga/SagaResources.xaml]: @distantcam This is done with the latest push. Please review.

Comment (distantcam) [inline src/ServiceInsight/Saga/SagaResources.xaml]: I'm not a fan of the new name either. What if the style changes to bold?

I prefer a functional name, like `SagaValueTextStyle`. Thoughts @adamralph?

Comment (HEskandari) [inline src/ServiceInsight/Saga/SagaResources.xaml]: If it changes to bold, should we question it then and rename accordingly? I'm open to other names but this is not a generic text style so don't think `SagaValueTextStyle` would be a good choice.

Comment (adamralph) [inline src/ServiceInsight/Saga/SagaResources.xaml]: I prefer a functional name too. I guess we can always refactor later if we feel strongly enough.

**Pull request merged**